code/ExercisesCheatSheet_All.md

```diff
@@ -169,6 +169,25 @@ Usually the program works fine, and valgrind is happy. The race condition is ind
 The title string is just deleted too early.
 This exercise should be skipped if time is missing
 
+### `clang-format`
+
+The goal is to understand that automatic code formatting is easy and can be customized
+
+Steps:
+
+1. Go to literally any example, which has no changes yet (we are going to revert it later)
```

Comment:
Maybe hint to a "good" one, with not too much code and interesting changes to be seen (as we do not use clang format currently in the CI)